--- OpenNI-OpenNI-1e9524f/Platform/Linux/Build/Modules/nimCodecs/Makefile.orig	2014-07-27 09:25:00.510513520 +0200
+++ OpenNI-OpenNI-1e9524f/Platform/Linux/Build/Modules/nimCodecs/Makefile	2014-07-27 09:26:03.623844205 +0200
@@ -2,15 +2,13 @@
 
 INC_DIRS = \
 	../../../../../Include \
-	../../../../../Source \
-	../../../../../Externals/LibJPEG
+	../../../../../Source
 
 SRC_FILES = \
-	../../../../../Source/Modules/nimCodecs/*.cpp \
-	../../../../../Externals/LibJPEG/*.c
+	../../../../../Source/Modules/nimCodecs/*.cpp
 
 LIB_NAME = nimCodecs
-USED_LIBS = OpenNI
+USED_LIBS = OpenNI jpeg
 
 include ../../Common/CommonCppMakefile
 
